test/parallel_exec.jl

```diff
@@ -820,7 +821,12 @@ if DoFullTest
     end
     sleep(0.5)  # Give some time for the above error to be printed
 
-@unix_only begin
+    # github PR #14456
+    for n = 1:10^6
+        fetch(@spawnat myid() myid())
```

Comment:
This test had been moved out of JUILIA_TESTFULL. Did you intentionally add it back with a larger loop count? I agree with the move, just making sure that it was intentional.